practice-one/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = (props) => {
```

Comment:
can use const Button = ({
type,
    children,
    width,
    ...
}) => {

Reply:
Yes, I changed it as your comment
![image](https://github.com/datdragon777/React-Training/assets/74806826/30a828e3-38ca-43b2-bf5b-cd7bf1ed4bb7)